torch/onnx/__init__.py

```diff
@@ -9,7 +9,7 @@
 # TODO: Update these variables when there
 # is a new ir_version and producer_version
 # and use these values in the exporter
-ir_version = 4
+ir_version = 6
 producer_name = "pytorch"
 producer_version = "1.3"
```

Comment:
@BowenBao - IR version 6 is only supported in ONNX 1.5 which is opset 11. We are releasing Opset 11 support in the upcoming PyTorch 1.4. Should we update the `producer_version` to 1.4 as well.

Reply:
updated